reference/7.2/Microsoft.PowerShell.Core/About/about_PowerShell_Config.md

```diff
@@ -39,358 +72,380 @@ installation.
+- `EnableConsoleSessionConfiguration` - to enable session configurations, set
+  the value to `true`.
 
-Sets the identity name used to write to the system log. The default value is
-"powershell".
+- `ConsoleSessionConfigurationName` - Specifies the name of configuration
+  endpoint in which PowerShell is run. This can be any endpoint registered on
+  the local machine including the default PowerShell remoting endpoints or a
+  custom endpoint having specific user role capabilities.
 
-```Schema
-"LogIdentity": "<log-identity>"
+```json
+{
+  "ConsoleSessionConfiguration": {
+    "EnableConsoleSessionConfiguration": false,
+    "ConsoleSessionConfigurationName" : []
+  }
+}
 ```
 
-Where:
+For more information, see [about_Session_Configurations][08].
 
-- `<log-identity>` is the string identity that PowerShell should use
-  for writing to syslog.
+### ExperimentalFeatures
 
-> [!NOTE]
-> You may want to have different **LogIdentity** values for each different
-> instance of PowerShell you have installed.
+The names of the experimental features to enable in PowerShell. The default
+value is an empty array.
 
-In this example, we are configuring the **LogIdentity** for a preview release
-of PowerShell.
+The following example enables the **PSCommandNotFoundSuggestion** and
+**PSSubsystemPluginModel** experimental features when PowerShell starts up.
+
+Example:
 
 ```json
 {
-  "LogIdentity": "powershell-preview"
+  "ExperimentalFeatures": [
+    "PSCommandNotFoundSuggestion",
+    "PSSubsystemPluginModel"
+  ]
 }
 ```
 
-### LogLevel
+For more information on experimental features, see
+[Using experimental features][05].
 
-> [!IMPORTANT]
-> This setting can only be used in macOS and Linux.
+### ModuleLogging
 
-Specifies the minimum severity level at which PowerShell should log.
+This setting controls the behavior of logging for PowerShell modules. The
+setting contains two subkeys:
 
-```Schema
-"LogLevel": "<log-level>|default"
-```
+- `EnableModuleLogging` - to enable session configurations, set the value to
+  `true`. When enabled, pipeline execution events for members of the specified
+  modules are recorded in the PowerShell log files.
+- `ModuleNames` - Specifies the name of the modules that should be logged.
 
-Where:
+Example:
 
-- `<log-level>` is one of:
-  - Always
-  - Critical
-  - Error
-  - Warning
-  - Informational
-  - Verbose
-  - Debug
+```json
+{
+  "ModuleLogging": {
+    "EnableModuleLogging": true,
+    "ModuleNames" : [
+        "PSReadLine",
+        "PowerShellGet"
+    ]
+  }
+}
+```
 
-> [!NOTE]
-> Setting a the log level enables all log levels above it.
+### ProtectedEventLogging
+
+This setting lets you configure Protected Event Logging. The setting contains
+two subkeys:
 
-Setting this setting to **default** will be interpreted as the default value.
-The default value is **Informational**.
+- `EnableProtectedEventLogging` - If you enable this policy setting, components
```

Comment:
Do we have documentation about setting this up anywhere? The certificate in the example is a name but it's not clear where this certificate is meant to be stored.